seccon23-rev-perfect-blu/index.md

```diff
@@ -0,0 +1,403 @@
+---
+title: "seccon23 rev/Perfect Blu"
+publishDate: "18 Sep 2023"
+description: "Author: es3n1n"
+tags: ["rev", "seccon23"]
+---
+
+
+#### Description
+
+Perfect Blu (135 pt)
+
+No, I'm real!
+
+[perfect-blu.tar.gz](https://score.quals.seccon.jp/api/download?key=quals2023%2FPerfect_Blu%2Fperfect-blu.tar.gz) 367b6ed67dda0afbbc975ee70ee946b4c7bf9268
+
+
+#### Overview
+
+Once I opened the downloaded `.tar.gz` file I observed that there was a `.iso` file inside and once I mounted it I saw that it had a file structure that looked 
+like the fs structure of a DVD. So what I did is I dragged this `.iso` file to the VLC and was very surprised.
+
+![1](./img/1.png)
+
+As you probably already figured, this task is a DVD image that has an interactive menu where you need to enter the flag, press on the `CHECK` button and, depending on the flag it would, either show a `SUCCESS` or a `WRONG...` message.
```

Comment:
_As you probably already figured_ - incorrect. better use one of:
- as you have probably already figured out
- as you might have guessed
(or variations)

_that has_ - maybe "with"?
_where you need_ - maybe "asking you"?

_press on the..._ - better split sentences. also extra `on`
_depending on the flag it would_ - what

```
As you have probably already figured out, this task is a DVD image with an interactive menu asking you to enter the flag.
Pressing the `CHECK` button shows either `SUCCESS` or `WRONG...` message, depending on the flag entered
```